tests_integration/test_nativeapp.py

```diff
@@ -496,3 +496,63 @@ def test_nativeapp_init_from_repo_with_single_template(
             assert result.exit_code == 0
         finally:
             single_template_repo.close()
+
+
+# Tests a simple flow of executing "snow app deploy", verifying that an application package was created, and an application was not
+@pytest.mark.integration
+def test_nativeapp_init_deploy(
```

Comment:
This is great -- can you also test that `list @stage` shows a `manifest.yml` at the root? I think that will be far enough to ensure that this is covering the contract.